recipe/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 1
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
```

Comment:
@marscher I am just curious what is your motivation for this as the byte code is in most of the times desirable.